index.js

```diff
@@ -5,6 +5,8 @@ exports.tzdata = tzdata;
 var _Date = Date;
 exports._Date = _Date;
 
+var MockDateOptions = {};
```

Comment:
Try running `npm run-script lint`, I suspect eslint will complain about this (expects only classes to be Capitalized), maybe call it `mockDateOptions` instead to match the style of the rest of the file.

Reply:
I did run lint with `--fix` to fix up whatever errors I had. Easy to rename if desired.